server/src/game/event/on_grave_added.rs

```diff
@@ -0,0 +1,17 @@
+use crate::game::{grave::Grave, Game};
+
+
+pub struct OnGraveAdded{
+    pub grave: Grave
```

Comment:
Why public? There's a `new` function you can use

Reply:
fixed